tests/functional/aws-node-sdk/test/bucket/aclUsingPredefinedGroups.js

```diff
@@ -1,33 +1,84 @@
+const {
+    CreateBucketCommand,
+    PutObjectCommand,
+    PutBucketAclCommand,
+    ListObjectsV2Command,
+    PutObjectAclCommand,
+    GetObjectCommand,
+    GetBucketAclCommand,
+    GetObjectAclCommand,
+    DeleteObjectCommand,
+} = require('@aws-sdk/client-s3');
 const { errorInstances } = require('arsenal');
-
 const withV4 = require('../support/withV4');
 const BucketUtility = require('../../lib/utility/bucket-util');
 const constants = require('../../../../../constants');
-const { VALIDATE_CREDENTIALS, SIGN } = AWS.EventListeners.Core;
+
 const itSkipIfE2E = process.env.S3_END_TO_END ? it.skip : it;
 const describeSkipIfE2E = process.env.S3_END_TO_END ? describe.skip : describe;
 
 withV4(sigCfg => {
     const ownerAccountBucketUtil = new BucketUtility('default', sigCfg);
     const otherAccountBucketUtil = new BucketUtility('lisa', sigCfg);
     const s3 = ownerAccountBucketUtil.s3;
-
     const testBucket = 'predefined-groups-bucket';
     const testKey = '0.txt';
     const ownerObjKey = 'account.txt';
     const testBody = '000';
 
-    function awsRequest(auth, operation, params, callback) {
+    async function awsRequest(auth, Operation, params) {
         if (auth) {
-            otherAccountBucketUtil.s3[operation](params, callback);
+            return await otherAccountBucketUtil.s3.send(new Operation(params));
         } else {
-            const bucketUtil = new BucketUtility('default', sigCfg);
-            const request = bucketUtil.s3[operation](params);
-            request.removeListener('validate', VALIDATE_CREDENTIALS);
-            request.removeListener('sign', SIGN);
-            request.send(callback);
+            // Create the command 
+            const command = new Operation(params);
+            
+            // Create unsigned client
+            const unsignedClient = new BucketUtility('default', {
+                ...sigCfg,
+                credentials: { accessKeyId: '', secretAccessKey: '' },
+                forcePathStyle: true,
+                signer: { sign: async request => request },
+            });
+
+            // Replace awsAuthMiddleware with a no-op middleware to skip signing
+            unsignedClient.s3.middlewareStack.use({
+                name: 'noAuthMiddleware',
+                step: 'serialize',
+                priority: 'high',
+                override: true,
+                tags: ['S3', 'NO_AUTH'],
+                applyToStack: stack => {
+                    stack.addRelativeTo(
+                        next => async args => {
+                            // Ensure no auth headers are added
+                            if (args.request && args.request.headers) {
+                                // eslint-disable-next-line no-param-reassign
+                                delete args.request.headers.Authorization;
+                                // eslint-disable-next-line no-param-reassign
+                                delete args.request.headers.authorization;
```

Comment:
Usually the headers always are lowercase in nodejs, so probably the one above can be dropped?